test/gtest/utils/test_ep_map.cc

```diff
@@ -131,7 +131,7 @@ class test_ep_map_inv : public test_ep_map {
     void check_inv(EpMap map)
     {
         ucc_ep_map_t inv;
-        EXPECT_EQ(UCC_OK, ucc_ep_map_create_inverse(map.map, &inv));
+        EXPECT_EQ(UCC_OK, ucc_ep_map_create_inverse(map.map, &inv, 0));
```

Comment:
what about adding test for the case where reordering is used ?

Reply:
reordering is used by default on the collective level, so gtests running ring alg are testing it. 
In this test we are just checking that an inverse map works, adding reordering flag is not really relevant here in my opinion as it won't test any additional functionality of the map.